### PR TITLE
chore: remove samples

### PR DESCRIPTION
**Pull request by meredithslota**
Samples have migrated here: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/scheduler/snippets Do not merge until snippetbot is happy.

### CODE REVIEW

**Review by cojenco — APPROVED** [review]
LGTM, thank you!

**Comment by cojenco** [comment]
I checked earlier today and snippetbot has picked up the migration. Ready to merge whenever samples owners approve :)

File: samples/snippets/README.md
Comment (dandhlee) [inline samples/snippets/README.md:1]: Could you add a snippet that's something along the lines of
```
The samples have moved to a new location: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/scheduler/snippets
```

**Review by dandhlee — APPROVED** [review]
LGTM, one comment.

File: samples/snippets/README.md
Comment (meredithslota) [reply samples/snippets/README.md:1]: Done, see latest commit. Thanks for flagging!

**Pull request merged**